crates/re_types_builder/src/codegen/rust/api.rs

```diff
@@ -355,6 +355,44 @@ fn quote_struct(
 
     let quoted_builder = quote_builder_from_obj(obj);
 
+    let quoted_heap_size_bytes =
+        if is_tuple_struct_from_obj(obj) && obj.fields[0].has_attr(crate::ATTR_RUST_SERDE_TYPE) {
```

Comment:
Even if it's just a hack, let's do a `iter.any()`, that should hold a bit better